app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,26 @@
+class CommentsController < ApplicationController
+  def new
+    @user = User.find(params[:user_id])
+    @post = Post.find(params[:post_id])
+    @comment = @post.comments.build
+  end
+
+  def create
+    @user = User.find(params[:user_id])
+    @post = Post.find(params[:post_id])
+    @comment = @post.comments.build(comment_params)
+    @comment.user_id = @user.id
```

Comment:
- [ ] According to the project requirement, a comment should be created on behalf of the currently logged in user. Therefore I suggest you make user of the `current_user` method created in the the Application Controller to create a new comment.
```rb
@user = current_user
```